api_loader.go

```diff
@@ -243,9 +243,8 @@ func processSpec(spec *APISpec, apisByListen map[string]int,
 	}
 
 	// Already vetted
-	remote, _ := url.Parse(spec.Proxy.TargetURL)
+	spec.target, _ = url.Parse(spec.Proxy.TargetURL)
```

Comment:
I know it is not your code, but still, don't you think it can cause a data race issues because spec object is shared across requests? In most cases, it will always be single value, but if there is lot of upstream rewrites logic it potentially looks dangerous.

Reply:
I don't know what you mean. This field is only written once when loading the spec. And it is read any number of times later. There is no possibility for a data race here.

Although, looking into this made me realise that this field is useless, as we only use it like a regular variable. So I'll simply make it one.

Reply:
Err, scratch that, it is used, I just misused grep horribly.

My point above still stands. If you can reason the existance of a race, feel free to open an issue (with a race detector error, preferrably). But I don't see the point of making this PR bigger, as it is an unrelated change/fix.

Reply:
Yeah, I did not mean that this issue should be in the context of this PR. Just wanted your thoughts on it.